configurate.go

```diff
@@ -30,8 +30,9 @@ func Init(path string) (*viper.Viper, error) {
 	return cfg, nil
 }
 
-// InitDefaults initializes the underlying config, using defaults for any unspecified configuration settings.
-func InitDefaults(path, defaultConfig string) (*viper.Viper, error) {
+// InitDefaultsReader initializes the underlying config from a file, using defaults for any unspecified configuration
+// settings.
+func InitDefaultsF(f *os.File, defaultConfig string) (*viper.Viper, error) {
```

Comment:
I think you can change the "f" parameter to be an io.Reader. The calling code should still be able to pass in an *os.File, I believe, since it satisfies the io.Reader interface. I checked the viper.MergeConfig() docs and it takes an io.Reader as well.

Reply:
Sounds good. I'll probably change the function name as well.